zookeeper-server/src/main/java/org/apache/zookeeper/Watcher.java

```diff
@@ -35,34 +35,33 @@
      * This interface defines the possible states an Event may represent
      */
     @InterfaceAudience.Public
-    public interface Event {
+    interface Event {
+
         /**
          * Enumeration of states the ZooKeeper may be at the event
          */
         @InterfaceAudience.Public
-        public enum KeeperState {
+        enum KeeperState {
             /** Unused, this state is never generated by the server */
-            @Deprecated
-            Unknown (-1),
+            @Deprecated Unknown(-1),
```

Comment:
Is this intended that we want to put annotation and annotated in the same line?

Reply:
Nope. Reverted.